package-lock.json

```diff
@@ -24718,7 +24719,7 @@
 					"dependencies": {
 						"kind-of": {
 							"version": "2.0.1",
-							"resolved": "https://registry.npmjs.org/kind-of/-/kind-of-2.0.1.tgz",
+							"resolved": "http://registry.npmjs.org/kind-of/-/kind-of-2.0.1.tgz",
```

Comment:
When I run `npm install` while on this branch, I have local changes which apply this back to the original:

```diff
diff --git a/package-lock.json b/package-lock.json
index faa74e088c..f7c0903b36 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -24685,7 +24685,7 @@
 			"dependencies": {
 				"clone-deep": {
 					"version": "0.2.4",
-					"resolved": "http://registry.npmjs.org/clone-deep/-/clone-deep-0.2.4.tgz",
+					"resolved": "https://registry.npmjs.org/clone-deep/-/clone-deep-0.2.4.tgz",
 					"integrity": "sha1-TnPdCen7lxzDhnDF3O2cGJZIHMY=",
 					"dev": true,
 					"requires": {
@@ -24719,7 +24719,7 @@
 					"dependencies": {
 						"kind-of": {
 							"version": "2.0.1",
-							"resolved": "http://registry.npmjs.org/kind-of/-/kind-of-2.0.1.tgz",
+							"resolved": "https://registry.npmjs.org/kind-of/-/kind-of-2.0.1.tgz",
 							"integrity": "sha1-AY7HpM5+OobLkUG+UZ0kyPqpgbU=",
 							"dev": true,
 							"requires": {

```

Related to: https://github.com/WordPress/gutenberg/pull/19680#pullrequestreview-344719610

I'm using latest version of NPM (6.13.6).

Reply:
Can you push them?

Reply:
This happens often when contributors merge PRs with different NPM versions.